src/workerd/io/BUILD.bazel

```diff
@@ -50,6 +50,7 @@ wd_cc_library(
         "io-context.c++",
         "io-own.c++",
         "io-util.c++",
+        "trace-stream.c++",
```

Comment:
In the interest of not making the io target more bloated, this should be a separate target – just needs to depend on `:io` and completion-membrane, and the server needs to depend on the new target too.